stylesheet/style.scss

```diff
@@ -0,0 +1,641 @@
+            .table-row {
+                display: flex;
+                background-color: #1B1B1B;
+                padding: 1.0625rem 3.625rem 1rem 3rem;
+                margin-bottom: 1.875rem;
+                @media screen and (max-width: 1280px) {
+                    padding: .8125rem 2.0625rem;
+                    margin-bottom: 1.3125rem;
+                }
+                @media screen and (max-width: 1024px) {
+                    padding: .9375rem 2.3125rem;
+                    margin-bottom: 1rem;
+                }
+                @media screen and (max-width: 991px) {
+                    padding: .9375rem 2.0625rem;
+                }
+                @media screen and (max-width: 768px) {
+                    padding: .75rem 1.5625rem;
+                    margin-bottom: .75rem;
+                }
+                @media screen and (max-width: 575px) {
+                    padding: .8125rem 1.875rem;
+                }
+                @media screen and (max-width: 480px) {
+                    padding: .625rem 1.25rem;
+                    margin-bottom: .5625rem;
+                }
+                .table-cell {
+                    font-size: 1.5rem;
+                    font-weight: 700;
+                    @media screen and (max-width: 1280px) {
+                        font-size: 1.25rem;
+                    }
+                    @media screen and (max-width: 1024px) {
+                        font-size: 1.125rem;
+                    }
+                    @media screen and (max-width: 768px) {
+                        font-size: 1rem;
+                    }
+                    @media screen and (max-width: 575px) {
+                        font-size: 1.25rem;
+                    }
+                    @media screen and (max-width: 480px) {
+                        font-size: 1.125rem;
+                    }
+                    @media screen and (max-width:375px) {
+                        font-size: .875rem;
+                    }
+                    &:first-child {
+                        width: 3.5rem;
+                        @media screen and (max-width: 1280px) {
+                            width: 2.875rem;
+                        }
+                        @media screen and (max-width:1024px) {
+                            width: 2.625rem;
+                        }
+                        @media screen and (max-width: 768px) {
+                            width: 2.3125rem;
+                        }
+                        @media screen and (max-width: 575px) {
+                            width: 2.875rem;
+                        } 
+                        @media screen and (max-width:480px) {
+                            width: 2.625rem;
+                        } 
+                        @media screen and (max-width: 375px) {
+                            width: 2.3125rem;
+                        }
+                    }
+                    &:nth-child(2) {
```

Comment:
rethink and refactor

Reply:
to avoid the repetition of the code, common properties of table head and table body cell are written once now by targeting the head and body with common class